### ISSUE TITLE
[1.16.5] Incompatibility with Rubidium

### ISSUE THREAD

**Issue by SammyWhamy** [octarine-noise/BetterFoliage#435]
When using BetterFoliage in combination with Rubidium, the game will crash while loading.
It would be amazing if this would be fixed, as BetterFoliage is an amazing mod, however in large modpacks its highly preferred to use Rubidium for greater performance.

Crash log:
[crash-2022-06-29_01.38.10-fml.txt](https://github.com/octarine-noise/BetterFoliage/files/9006126/crash-2022-06-29_01.38.10-fml.txt)
 
 p.s. Rubidium Extras and Rubidium Dynamic Lights work without issues, only the base Rubidium mod causes a crash.

**Comment by grimmspector** [comment]
Yup, can confirm, I'm also having htis issue :( I'm guessing it's not getting fixed given how old this issue is??
